backend/src/main/java/com/hyooun/wilson/review/adapter/in/web/ReviewRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.hyooun.wilson.review.adapter.in.web;
+
+import com.hyooun.wilson.review.domain.Change;
+
+import java.util.List;
+
+public record ReviewRequest(
+    Long projectId,
+    String prTitle,
+    String prBody,
+    List<ChangeInfo> changes
+) {
+    public record ChangeInfo(
+        String filePath,
+        Long lineNumber,
+        String oldCode,
+        String newCode,
+        String type
+    ) {}
+}
```

Comment:
ReviewRequest 클래스에 ChangeInfo 내부 클래스를 추가해주세요.

```suggestion
public record ChangeInfo(String filePath, Long lineNumber, String oldCode, String newCode, String type) {}
```